Add plugin: Easy Test

forrest1398 opened the pull request:
# I am submitting a new Community Plugin

## Repo URL

<!--- Paste a link to your repo here for easy access -->
Link to my plugin: https://github.com/forrest1398/obsidian-easyTest-plugin/tree/1.0.0

## Release Checklist
- [x] I have tested the plugin on
  - [ ]  Windows
  - [x]  macOS
  - [ ]  Linux
  - [ ]  Android _(if applicable)_
  - [ ]  iOS _(if applicable)_
- [x] My GitHub release contains all required files (as individual files, not just in the source.zip / source.tar.gz)
  - [x] `main.js`
  - [x] `manifest.json`
  - [x] `styles.css` _(optional)_
- [x] GitHub release name matches the exact version number specified in my manifest.json (_**Note:** Use the exact version number, don't include a prefix `v`_)
- [x] The `id` in my `manifest.json` matches the `id` in the `community-plugins.json` file.
- [x] My README.md describes the plugin's purpose and provides clear usage instructions.
- [x] I have read the developer policies at https://docs.obsidian.md/Developer+policies, and have assessed my plugins's adherence to these policies.
- [x] I have read the tips in https://docs.obsidian.md/Plugins/Releasing/Plugin+guidelines and have self-reviewed my plugin to avoid these common pitfalls.
- [x] I have added a license in the LICENSE file.
- [x] My project respects and is compatible with the original license of any code from other plugins that I'm using.
      I have given proper attribution to these other projects in my `README.md`.

forrest1398 commented:
Hello Obsidian team, I have a question.

I updated the review comments and completed the push, but it seems that the changes won't be reflected unless I modify the release. 
So, I deleted release 1.0.0 and created a new release 1.0.0 with the updated commit. Would this cause any issues?

joethei commented:
[this.addSettingTab(new EasyTestPluginSettingTab(this.app, this));](https://github.com/forrest1398/obsidian-easyTest-plugin/blob/67c4de192cef445dfb1be7d712ce67640f03c538/main.ts#L40)
Since your settings tab is currently offering no settings, remove this.

["Open Test",](https://github.com/forrest1398/obsidian-easyTest-plugin/blob/67c4de192cef445dfb1be7d712ce67640f03c538/main.ts#L45)
[Use sentence case in UI](https://docs.obsidian.md/Plugins/Releasing/Plugin+guidelines#Use+sentence+case+in+UI)

[hotkeys: [{ modifiers: ["Mod", "Shift"], key: "i" }],](https://github.com/forrest1398/obsidian-easyTest-plugin/blob/67c4de192cef445dfb1be7d712ce67640f03c538/main.ts#L70)
You should not provide a default hotkey. The reason being that it's easy to pick a hotkey that a user already has configured and they could get confused when that key doesn't do what they expect. Also, it's hard choosing a safe default hotkey that's available for all operating systems & does not conflict with existing plugins.
This hotkey for example is already used by the developer tools.

forrest1398 commented:
Hello @joethei 
Thank you for the review. As per the review comments, I have removed the settings tab and the hotkey. 
Additionally, I have also added the new features to the release.

joethei commented:
["Open Test",](https://github.com/forrest1398/obsidian-easyTest-plugin/blob/a4723b2ae04751b3b2663854e8d20b6a4fdd4499/main.ts#L26)
[Use sentence case in UI](https://docs.obsidian.md/Plugins/Releasing/Plugin+guidelines#Use+sentence+case+in+UI)

forrest1398 commented:
Hello @joethei �
I'm sorry , i missed that item.
Now i uptated that
